.hooks/plugin-warning-logger.js

```diff
@@ -5,10 +5,10 @@ module.exports = {
       validateProject() {
         const nodeVersion = process.version
 
-        if (!nodeVersion.startsWith('v16.')) {
+        if (!nodeVersion.startsWith('v18.')) {
           console.error('\x1b[1m' /* bold */ + '╔'.padEnd(79, '═') + '╗')
-          console.error(`║ Must be running node 16, you have ${nodeVersion}. One way to fix (Mac/Homebrew):`.padEnd(79) + '║')
-          console.error('║ $ brew install node@16; brew link node@16 --force --overwrite'.padEnd(79) + '║')
+          console.error(`║ Must be running node 18, you have ${nodeVersion}`.padEnd(79) + '║')
+          console.error('║ See https://nodejs.org/en/download/ for installation options'.padEnd(79) + '║')
```

Comment:
@nawatts I missed this one earlier. Why did we get rid of the message with the `brew` commands. I've found them helpful.

Reply:
It seems overly prescriptive. Not everyone uses macOS and Homebrew. Installation using Homebrew is covered in the linked docs ("Installing Node.js via package manager").

Also, since Node 18 is the current version, there is no `node@18` formula. Using `brew install node` here would become incorrect whenever Node 19 is released.